var/spack/repos/builtin/packages/py-zc-buildout/package.py

```diff
@@ -9,7 +9,7 @@
 class PyZcBuildout(PythonPackage):
     """System for managing development buildouts"""
 
-    homepage = "http://buildout.org/"
+    homepage = "https://pypi.org/project/zc.buildout/"
```

Comment:
You can remove this line, it's the default set by `pypi`

Reply:
This should be fixed.